lib/ws_servers/api/algos/util/validate_ao.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const _isFunction = require('lodash/isFunction')
+
+module.exports = (aoHost, aoID, payload) => {
+  const ao = aoHost.getAO(aoID)
+
+  if (!ao) {
+    return `Unknown algo order ID: ${aoID}`
+  }
+
+  const { meta = {} } = ao
+  const { validateParams, processParams } = meta
+
+  const params = _isFunction(processParams)
+    ? processParams(payload)
+    : { ...payload }
+
+  if (_isFunction(validateParams)) {
```

Comment:
💅🏻 we can return earlier if `validateParams` is not a function thus avoiding the `processParams` to be called

Reply:
fixed!